homeassistant/components/light/insteon_plm.py

```diff
@@ -61,59 +60,65 @@ def should_poll(self):
     @property
     def address(self):
         """Return the address of the node."""
-        return self._address
+        return self._device.address.human
 
     @property
-    def name(self):
+    def id(self):
         """Return the name of the node."""
-        return self._name
+        return self._device.id
+
+    @property
+    def name(self):
+        """Return the name of the node. (used for Entity_ID)"""
+        if self._newnames:
+            return self._device.id + '_' + self._state.name
+        else:
+            if self._state.group == 0x01:
+                return self._device.id
+            else:
+                return self._device.id+'_'+str(self._state.group)
 
     @property
     def brightness(self):
         """Return the brightness of this light between 0..255."""
-        onlevel = self._plm.get_device_attr(self._address, 'onlevel')
-        _LOGGER.debug("on level for %s is %s", self._address, onlevel)
+        onlevel = self._state.value
+        _LOGGER.debug("on level for %s is %s", self._device.address, onlevel)
         return int(onlevel)
 
     @property
     def is_on(self):
         """Return the boolean response if the node is on."""
-        onlevel = self._plm.get_device_attr(self._address, 'onlevel')
-        _LOGGER.debug("on level for %s is %s", self._address, onlevel)
-        return bool(onlevel)
+        _LOGGER.debug("on level for %s is %s",
+                      self._device.id, self.brightness)
+        return bool(self.brightness)
 
     @property
     def supported_features(self):
         """Flag supported features."""
-        if self._dimmable:
-            return SUPPORT_BRIGHTNESS
+        return SUPPORT_BRIGHTNESS
 
     @property
     def device_state_attributes(self):
         """Provide attributes for display on device card."""
         insteon_plm = get_component('insteon_plm')
-        return insteon_plm.common_attributes(self)
-
-    def get_attr(self, key):
-        """Return specified attribute for this device."""
-        return self._plm.get_device_attr(self.address, key)
+        return insteon_plm.common_attributes(self._device, self._state)
```

Comment:
What is part of these? We should not store any static information like manufacturer in the state attributes

Reply:
I am sorry but I don't understand this one. `insteon_plm.common_attributes` has not changed meaningfully from the prior code. When I look at other platforms like Z-Wave they seem to be using this function the same way. Can you explain a little more?